### PR TITLE
Add new Elise, 65% Keyboard ANSI / ISO 

### PR DESCRIPTION
**Pull request by ghostseven**
## Description

Add a new keyboard by Draytronics. Elise 65% keyboard, supporting ANSI / ISO layouts with USB-C, under-glow, ESD protection and physical / dimensional compatibility with XD68 cases and plates. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ x ] Keyboard (addition or update)
- [ x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Comment by ghostseven** [comment]
Is there anything I need to do with regards to getting this over the line and merged?

File: keyboards/draytronics/elise/readme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+![elise](https://www.draytronics.co.uk/wp-content/uploads/2021/04/repository-open-graph-template.png)
+
+A open source, low cost 65% keyboard PCB with USB-C, ESD protection, RGB underglow and XD68 case / plate compatability.  Designed in the UK. 
+More info / sales / PCB designs available at [draytronics.co.uk/elise](https://draytronics.co.uk)
+
+* Keyboard Maintainer: [Blake Drayson](https://github.com/ghostseven)
+* Hardware Supported: ELISE PCB / ATMega32U4
+* Hardware Availability: [draytronics.co.uk](https://draytronics.co.uk)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/draytronics/elise/readme.md:12]: Could you add instructions on how to enter the bootloader to flash new firmware?

File: keyboards/draytronics/elise/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+#define _BL 0
+#define _FL 1
```
Comment (drashna) [inline keyboards/draytronics/elise/keymaps/default/keymap.c:21]: In general, we highly recommend using an enum for the layer naming, as it's easier to maintain, long term. 
```suggestion
enum custom_layers {
    _BL,
    _FL,
};
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/draytronics/elise/rules.mk [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
```
Comment (fauxpark) [inline keyboards/draytronics/elise/rules.mk:22]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/draytronics/elise/rules.mk [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# MCU name
+MCU = atmega32u4
+BOOTLOADER = atmel-dfu
```
Comment (fauxpark) [inline keyboards/draytronics/elise/rules.mk:3]:
```suggestion

# Bootloader selection
BOOTLOADER = atmel-dfu
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/draytronics/elise/readme.md (hunk 1 above)
Comment (ghostseven) [reply keyboards/draytronics/elise/readme.md:19]: Instructions providing various methods to enter DFU mode have now been added to the readme.md

**Review by drashna — APPROVED** [review]

File: keyboards/draytronics/elise/rules.mk [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# QMK Build Options
```
Comment (fauxpark) [inline keyboards/draytronics/elise/rules.mk:7]:
```suggestion
# Build Options
```

File: keyboards/draytronics/elise/rules.mk [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
```
Comment (fauxpark) [inline keyboards/draytronics/elise/rules.mk:19]:
```suggestion
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**